stdlib/src/builtin/simd.mojo

```diff
@@ -2618,6 +2708,48 @@ struct SIMD[type: DType, size: Int](
         ](zero_simd, self, Int32(-shift))
 
 
```

Comment:
Yeah but since these functions are merely calling an intrinsic, marking them as always inline can save some IR generation (and we can't step into them anyways) I think. Not super important though, I won't waste your time trying to push this, it's more of a FYI.